### DISCUSSION TITLE
Is it possible to obtain a trainable parameters matrix for GCNConv?

### DISCUSSION THREAD

**Discussion by raythroughspace** [pyg-team/pytorch_geometric#7101] [General]
I am using a GCN with 1 layer to do a classification task. The GCN is successful in classifying my samples, is it possible to obtain the trainable parameters matrix (which is denoted $\theta$ in the GCNConv docs)? Specifically, I'm looking to use this matrix directly to determine which nodes in my graphs have the largest influence on the predictions, so it is important for me to know which trainable parameter is associated with which edges.

**Comment by rusty1s** [comment]
You can access it via `conv.lin.weight`.
